src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Usage/DetectPLINQNopsTest.cs

```diff
@@ -0,0 +1,183 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.NetCore.Analyzers.Usage;
+using Xunit;
+
+using VerifyCS = Test.Utilities.CSharpCodeFixVerifier<
+    Microsoft.NetCore.Analyzers.Usage.DetectPLINQNopsAnalyzer,
+    Microsoft.NetCore.Analyzers.Usage.DetectPLINQNopsFixer>;
+
+namespace Microsoft.CodeAnalysis.NetAnalyzers.UnitTests.Microsoft.NetCore.Analyzers.Usage
+{
+    public sealed class DetectPLINQNopsTest
```

Comment:
Please add tests for VB.NET.

Reply:
I do not know any VB.NET, this might take a while

Reply:
It's ok don't bother with it. I will try to provide them so we can check everything is working as planned.

Reply:
Thank you, I was just reading up on VB. Seems worthwile to learn the basics so this does not become a recurring thing

Reply:
There are online converters you could use. They are not perfect but they are working pretty well, that gives  90% of the job and it's also a good way to "learn".